store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/StoreReadRequest.kt

```diff
@@ -66,6 +68,16 @@ data class StoreReadRequest<out Key> private constructor(
             refresh = refresh
         )
 
+        /**
+         * Create a [StoreReadRequest] which will return data from memory/disk caches if present,
+         * otherwise will return [StoreReadResponse.NoNewData]
+         */
+        fun <Key> cacheOnly(key: Key) = StoreReadRequest(
```

Comment:
Thoughts on `localOnly` or `cacheOrSourceOfTruth` or `withoutFetcher`? My concern is users expecting `cacheOnly` to literally only hit cache

Reply:
I understand your reasoning, and I do agree, but the existing `cache` already suffers from that, it already is memory cache + SoT.

Reply:
> I understand your reasoning, and I do agree, but the existing cached already suffers from that, it already is memory cache + SoT.

Yeah, that's fair.

> Maybe cached should be renamed to localFirst?

I agree `cached` could be clarified, though want to avoid breaking changes.

I'm on board with `cacheOnly` 👍🏽

Reply:
Alright, should I revert 15c6f6327c52ec668d6cf009ae3b58192fa21a8d then?

> I agree cached could be clarified, though want to avoid breaking changes.

Of course, I was going to suggest adding `localFirst` as a new extension function and then deprecating `cached` for removal at a later date.

Reply:
No need to rename. I like `localOnly`. (I commented above before seeing the name change. I stamped after seeing the name change, though would have stamped `cacheOnly` too)